src/snowflake/cli/_plugins/nativeapp/manager.py

```diff
@@ -576,30 +580,36 @@ def create_app_package(self) -> None:
                 )
             )
 
-    def _expand_script_templates(
-        self, env: jinja2.Environment, jinja_context: dict[str, Any], scripts: List[str]
+    def _render_script_templates(
+        self,
+        render_from_str: Callable[[str, Dict[str, Any]], str],
+        jinja_context: dict[str, Any],
+        scripts: List[str],
     ) -> List[str]:
         """
         Input:
-        - env: Jinja2 environment
+        - render_from_str: function which renders a jinja template from a string and jinja context
         - jinja_context: a dictionary with the jinja context
-        - scripts: list of scripts that need to be expanded with Jinja
+        - scripts: list of script paths relative to the project root
         Returns:
-        - List of expanded scripts content.
+        - List of rendered scripts content
         Size of the return list is the same as the size of the input scripts list.
         """
         scripts_contents = []
         for relpath in scripts:
+            script_full_path = SecurePath(self.project_root) / relpath
             try:
-                template = env.get_template(relpath)
-                result = template.render(**jinja_context)
+                template_content = script_full_path.read_text(
+                    file_size_limit_mb=DEFAULT_SIZE_LIMIT_MB
```

Comment:
@sfc-gh-melnacouzi if we want to omit this check, we can use `SecurePath.UNLIMITED` here

Reply:
For SQL files, 128MB is a lot. It's probably ok to keep it here as long as the error message to the user is clear.